src/components/PatientBanner/patient-banner.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styles from './patient-banner.css';
+
+
+function PatientBanner(props) {
```

Comment:
Can you add some initial tests for this new component (e.g. rendering it with and without the attributes being present)?

Reply:
Done.  PTAL